File: malacprodavac/src/Backend/src/main/java/com/example/Backend/entity/DeliveryRequest.java
package com.example.Backend.entity;

import jakarta.persistence.*;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Entity
@Table(name = "deliveryrequest")
@Getter
@Setter
@NoArgsConstructor
public class DeliveryRequest {
    @Id
    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "hibernate_sequence")
    @Column(name = "id")
    private Long id;
    private Long sellerId;
    private Long delivererId;
    private String cityStart;
    private String cityEnd;
    private String date;
    private String time;
}

File: malacprodavac/src/Frontend/app/src/main/java/com/example/myapplication/models/DeliveryRequest.kt
package com.example.myapplication.models

data class DeliveryRequest (
    val id: Long,
    val image: String,
    val companyName: String,
    val firstNameUser: String,
    val lastNameUser: String,
    val cityStart: String,
    val cityEnd: String,
    val date: String,
    val time: String
)